test/dwd_dynam.jl

```diff
@@ -310,7 +310,76 @@ end
     @test readout(m, vcat(u1, u2, u3), nothing, 0) == [u1 + u2, u2, u3]
     @test eval_dynamics(m, vcat(u1, u2, u3), [x1, x2], nothing, 0) == vcat(x1, u2 + x1 + 2*u2 + u3, x2)
   end
-end
 
+  @testset "VectorInterface for InstantaneousContinuousMachine" begin
+    m = InstantaneousContinuousMachine{Float64,3}(
+      1, 3, 1,
+      (u, x, p, t) -> u*1.5, # dynamics
+      (u, x, p, t) -> u+x, # readout
+      [1 => 1]
+    )
+    
+    x1 = Float64.([1,2,3])
+    u1 = Float64.([4,5,6])
+    
+    @test readout(m, u1, x1, nothing, 0) == u1+x1
```

Comment:
Ah right, this should be `readout(m, u1, [x1], nothing, 0)`

Reply:
fixed.